Assets/AY_Scripts/ShellController.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ShellController : MonoBehaviour
+{
+    public int shellDamage;
+    public ParticleSystem explosion;
+    private int enemyLayer = 10;
+    private void OnCollisionEnter(Collision collision)
+    {
+        if (collision.gameObject.layer.Equals(enemyLayer))
+        {
+            collision.gameObject.GetComponent<EnemyController>().Damage(shellDamage);
```

Comment:
Have a separate Explosion controller and give it the point to play the explosion when an enemy gets hit with the bullet.